src/CalcViewModel/CalcViewModel.vcxproj

```diff
@@ -408,6 +408,23 @@
       <Project>{311e866d-8b93-4609-a691-265941fee101}</Project>
     </ProjectReference>
   </ItemGroup>
+  <ItemDefinitionGroup Condition="!Exists('DataLoaders\DataLoaderConstants.h')">
```

Comment:
Any reason this can't go inside the `Choose` as well?

Reply:
I did try it out and it seems that choose doesn't allow for ItemDefinitionGroup to be used inside.